lake/api/objectreader.go

```diff
@@ -0,0 +1,53 @@
+package api
```

Comment:
I think this should live in cmd/zed/manage or cmd/zed/manage/compact, at least until "zed manage" is more mature (and possibly forever since this doesn't feel like it'll get a lot of use).

Reply:
ok